Extensions/TweenBehavior/JsExtension.js

```diff
@@ -458,6 +638,71 @@ module.exports = {
       .getCodeExtraInformation()
       .setFunctionName('addVariableTween2');
 
+    behavior
+      .addScopedAction(
+        'AddObjectVariableTween3',
+        _('Tween a number in an object variable'),
+        _(
+          "Tweens an object variable's numeric value from its current value to a new one."
+        ),
+        _(
+          'Tween the variable _PARAM3_ of _PARAM0_ to _PARAM4_ with easing _PARAM5_ over _PARAM6_ seconds as _PARAM2_'
+        ),
+        _('Variables'),
+        'JsPlatform/Extensions/tween_behavior24.png',
+        'JsPlatform/Extensions/tween_behavior32.png'
+      )
+      .addParameter('object', _('Object'), '', false)
+      .addParameter('behavior', _('Behavior'), 'TweenBehavior', false)
+      .addParameter('identifier', _('Tween Identifier'), 'objectTween')
+      .addParameter('objectvar', _('Object variable'), '', false)
+      .addParameter('expression', _('To value'), '', false)
+      .addParameter('stringWithSelector', _('Easing'), easingChoices, false)
+      .setDefaultValue('linear')
+      .addParameter('expression', _('Duration (in seconds)'), '', false)
+      .addParameter(
+        'yesorno',
+        _('Destroy this object when tween finishes'),
+        '',
+        false
+      )
+      .setDefaultValue('no')
+      .getCodeExtraInformation()
+      .setFunctionName('addVariableTween3');
+
+      behavior
+        .addScopedAction(
+          'AddObjectValueTween',
+          _('Tween an object value'),
+          _('Tweens an object value that can be use with the object expression Tween::Value.'),
+          _(
+            'Tween the value of _PARAM0_ from _PARAM4_ to _PARAM5_ with easing _PARAM6_ over _PARAM7_ seconds as _PARAM2_'
```

Comment:
I think there's an offset issue with the params: PARAM4 parameter is the fifth one and corresponds to the "To value"